content/en/docs/ops/configuration/traffic-management/protocol-selection/index.md

```diff
@@ -63,3 +63,15 @@ spec:
   - number: 80
     name: http-web
 {{< /text >}}
+
+## Gateways
```

Comment:
This is a bit subtle, here is what I wanted to get across:

* Downsteam protocol and upstream protocol are unrelated by default for Gateway, but not for mesh
* This means you ought to configure the upstream protocol if you want HTTP/2, since otherwise we use HTTP/1
* If you really know what you are doing, you can set useClientProtocol, but should be advised that many clients WILL use http/2, even if you didn't want them to, due to ALPN negotiation.